steamclog/consumer-rules.pro

```diff
@@ -1,8 +1,9 @@
-# Required if you want to use Steamclog Redactable classes, and Proguard/R8 to obsfucate code.
+## Required if you want to use Steamclog Redactable classes, and Proguard/R8 to obsfucate code.
 -keep class * extends com.steamclock.steamclog.Redactable { *; }
-
-# Sample specific
 -keep public class * { # All public classes
     public static *; # All public static fields in those classes
     public protected abstract *(...); # All public or protected abstract methods in those classes
-}
\ No newline at end of file
+}
+
+# Required for run time verfication that CustomStackElementTag is correct
+-keep,allowoptimization class com.steamclock.steamclog.* { *; }
```

Comment:
I'm trying to make it so the private `logInternal` method name is not being changed, it's possible there is a better way to do this.